seed for offer model completed

minami-77 opened the pull request:
I added seed for offer model.

Munkleson commented:
Looks good! Just a quick one, we don't need the "created_at" and "updated_at" when creating a record, so we can get rid of it.

As a side note, could you possibly also make a loop for the offer seeding in case we want to have more displayed please?

minami-77 commented:
Thank you for checking, Alex! I added a loop for offers seeds and modified listings seeds as well. 
I also created users seeds in case we need them. I made 50 but we can change the number of seeds if we need more.

minami-77 commented:
delete the loop for user seed and add one user.
  first_name: "Bob",
  last_name: "Tanaka",
  email: "bob@email.com",
  username: "Bob",
  password: "123456"
